app/scripts/directives/mainSearchBoxHeight.dir.js

```diff
@@ -0,0 +1,26 @@
+'use strict';
+
+angular.module('yourStlCourts').directive('mainSearchBox', function ($window) {
+  return {
+    restrict: 'E',
+    replace: true,
+    transclude: true,
+    template: '<div id="mainSearchBox" class="accent" ng-transclude></div>',
+    scope: false,
+    link: function ($scope, element, attrs) {
+      angular.element($window).on('resize',function(){
+        resizeMainSearchBox();
+      });
+      resizeMainSearchBox(); //call it for initial setup
+
+      function resizeMainSearchBox(){
+        var newHeight = $(element).parentsUntil('body').parent().width()*.3125;
```

Comment:
Isn't this just the <html> tag? Could you just get the width of the window/document? 

Also what's with the ".3125" ? Seems kind of random. If this is indeed needed, could you define this variable with a name so it's clearer?